app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  has_many :likes, foreign_key: 'posts_id'
+  has_many :comments, foreign_key: 'posts_id'
+  belongs_to :author, class_name: 'User', dependent: :destroy, foreign_key: 'author_id'
+end
```

Comment:
Great job @eerapheal 🤝

- [ ] Kindly make sure the post model has a method that updates the posts counter for a user.
![post-counter](https://user-images.githubusercontent.com/63917725/215031352-a32b21c7-f120-42d1-af21-3ae98438da02.PNG)

- [ ] [Optional] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private. You may do this for the likes_counter and the comments_counter.
- [ ] Kindly make sure the Post model has a method that returns the 5 most recent comments for a given post.
![post-recent-comments](https://user-images.githubusercontent.com/63917725/215032176-ced0ecb9-5097-461d-9707-2641aa1616f5.PNG)